packages/components/boxes/Box.tsx

```diff
@@ -53,6 +53,7 @@ export type BoxStyle = Pick<
   | "top"
   | "bottom"
   | "flex"
+  | "zIndex"
```

Comment:
zIndex should be passed to the outer component IMO

Reply:
Yep i agree